src/foam/core/Validation.js

```diff
@@ -541,3 +541,26 @@ foam.CLASS({
     }
   ]
 });
+
+foam.CLASS({
```

Comment:
Should you also remove the validation from User.js and elsewhere as part of this PR?

Reply:
I did remove it. Not sure what you mean